pom.xml

```diff
@@ -164,16 +177,50 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>${checkstyle.version}</version>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>findbugs-maven-plugin</artifactId>
+          <!-- todo: migrate to spotbugs-maven-plugin as the findbugs plugin is no longer maintained -->
+          <version>3.0.5</version>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-pmd-plugin</artifactId>
+          <version>3.19.0</version>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>versions-maven-plugin</artifactId>
+          <version>2.13.0</version>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>taglist-maven-plugin</artifactId>
+          <version>3.0.0</version>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-changelog-plugin</artifactId>
+          <version>2.3</version>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-changes-plugin</artifactId>
+          <version>2.12.1</version>
+        </plugin>
       </plugins>
     </pluginManagement>
+
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>${java-compile.version}</source>
```

Comment:
why? should stay in the project IMHO

Reply:
have to admit I prefer the configuration to be colocalised instead of split accross section, it enables a better readability in editors and avoids to scroll to get it (hover gives the value to be concrete). The other advantage for newbies is it avoids to know all conventions and check the doc/plugin.xml to get the defaults. wdyt?

Reply:
I've made a habit of specifying it in the property format on all maven projects I work on. Done that way for years and although there is some knowledge newcomers may need, it's something worth knowing, especially as that property is now used by numerous plugins (even non-apache). I can put it back to being explicit but would at least use the standard property key.

Reply:
it is ok for me to use the property with the default name, no issue there but ensure the wiring is explicit please (funnily I used the exact same reasonning so hope both can converge ;))